### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MeH2607**
did something funny in the first method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ public class PoorlyWrittenCode {
 
     public static void main(String[] args) {
         int a = 10;
-        int b = 1;
+        int b = 0;
```
Comment (jonathanFS14) [inline README.md:7]: define variables more clearly, something like....

Comment (MeH2607) [reply README.md:7]: my bad

**Comment by jonathanFS14** [comment]
learn to code bruh

**Comment by MeH2607** [comment]
> 

loks good to me

**Pull request merged**